src/cirrus/docs/components/tasks/index.rst

```diff
@@ -193,14 +195,89 @@ Lambda and Batch
 ^^^^^^^^^^^^^^^^
 
 For tasks that should support both Lambda and Batch, run the ``create``
-command, this time using the options ``--has-lambda`` and ``--has-batch``::
+command, this time using the options ``--type lambda`` and ``--type batch``::
 
-    ❯ cirrus create task --has-lambda --has-batch <TaskName> "<task description>"
+    ❯ cirrus create task --type lambda --type batch <TaskName> "<task description>"
 
 This command does the same as both of the above ``create`` command examples, so
 the listed caveats of both apply here: ensure the handler code is completed,
 and the batch configuration is updated to match the task requirements.
 
+* :doc:`Conditionally Using Batch or Lambda <../workflows/batch>`
+
+Docker Image
+---------------
+
+Unfortuantely, the same Docker image cannot be used by both Batch and Lambda, as they
```

Comment:
I thought we had a way for tasks to `RunBatchAsLambda` and `RunLambdaAsBatch` but I can't seem to find it anymore.

Reply:
@matthewhanson updated one the examples to use a Choice, and that attempted to use the same image, which didn't work. Matt, do you know of what Dan's referencing?

Reply:
There used to be a Docker image used in a generic Batch task and that could be used to run any Lambda that was deployed as a standard Lambda package (ie zip file).   The task would take in the ARN of the Lambda, fetch the deploy package code, unzip, then run the included task.py file.   It was a way to just deploy a Lambda, and not have to deploy anything else.

The utility of this isn't really there anymore.  Most importantly, we now deploy Lambdas as images to begin with, so this same image should be runnable in either Batch or Lambda. (though right now there's an issue I think with how ENTRYPOINT is set that makes these images incompatible.

Also, while the original idea of this was useful for development, in production you end up fetching the deploy package over and over again, every time you run a job.

tl;dr - now with stac-task you develop and deploy a single image, which can then be used as a Lambda or Batch task.